### ISSUE TITLE
soft hyphen (00AD) is invisible

### ISSUE THREAD

**Issue by u-fischer** [IBM/plex#660]
The soft hyphen in the Plex fonts is invisible and has width zero. That is a problem in a Tagged PDF as soft hyphens should be used for  hyphenated words to improve accessibility and screen reading.  See also https://jkorpela.fi/shy.html#pdf for a discussion about the soft hyphen in general and its use in PDF specifically

It is also different to almost all other fonts (I tested some time ago more than a few hundred fonts).
Attached a PDF that shows the soft hyphen for latin modern, Arial and Plex.

[test-utf8.pdf](https://github.com/user-attachments/files/21614175/test-utf8.pdf)
